### ISSUE TITLE
missing javascript assets

### ISSUE THREAD

**Issue by stdweird** [hpcugent/vsc_user_docs#482]
```
GEThttp://hpcugent.github.io/vsc_user_docs/HPC/Gent/assets/javascripts/bundle.6c7ad80a.min.js
[HTTP/1.1 404 Not Found 147ms]

Loading failed for the <script> with source “http://hpcugent.github.io/vsc_user_docs/HPC/Gent/assets/javascripts/bundle.6c7ad80a.min.js”. Gent:1778:1
```

(from firefox inspect on http://hpcugent.github.io/vsc_user_docs/HPC/Gent/)

**Comment by stdweird** [comment]
not sure what meeds it though

**Comment by alvarosimon** [comment]
@stdweird indeed:

```
$ ls HPC/Gent/assets/javascripts/bundle.6c7ad80a.min.js
ls: cannot access 'HPC/Gent/assets/javascripts/bundle.6c7ad80a.min.js': No such file or directory

 $ ls HPC/Gent/Gent/assets/javascripts/bundle.6c7ad80a.min.js
HPC/Gent/Gent/assets/javascripts/bundle.6c7ad80a.min.js
```

This issue is fixed by this hack #483

**Comment by stdweird** [comment]
is fixed
